packaging/get_version.sh

```diff
@@ -0,0 +1,29 @@
+#! /bin/bash
+
+set -e
+
+OLD_PWD=$(pwd)
+SRC_DIR="$(dirname "$0")/.."
+
+cd "$SRC_DIR"
+
+VERSION="$(python setup.py --version)"
+DATE="$(date +%Y%m%d)"
+
+COMMIT_COUNT="$(git rev-list --count HEAD --)"
+GIT_REVISION="$(git rev-parse --short HEAD)"
+
+TAR_VERSION="${VERSION}dev${COMMIT_COUNT}git${GIT_REVISION}"
+
+RPM_VERSION="${VERSION}"
+RPM_RELEASE="0.${DATE}.${COMMIT_COUNT}git${GIT_REVISION}"
+
+RPM_DATE="$(LANG=C date +"%a %b %d %Y")"
+
+RPM_CHANGELOG="* ${RPM_DATE} N. N. - ${RPM_VERSION}-${RPM_RELEASE}"
```

Comment:
What does `N. N.` mean here?

Reply:
It is a placeholder name: https://en.wikipedia.org/wiki/Nomen_nescio